auditsweeper/auditsweeper.py

```
@@ -0,0 +1,87 @@
+    else:
+        print("One or more required environment variables are not set. Checking for local.properties...")
+        config = configparser.ConfigParser()
+        config_file = "local.properties"
+
+        if not os.path.exists(config_file):
+            print(f"Error: Required variables not set and '{config_file}' not found.")
+            sys.exit(1)
+
+        try:
+            # Read the properties file, assuming a single section
+            config.read_string(f"[DEFAULT]\n{open(config_file).read()}")
```

Ruff (0.16.3)

[warning] Use a context manager for opening files

(SIM115)